release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -905,6 +919,16 @@ In the table below, features are marked with the following statuses:
 |TP
 |GA
 
+|Cloud controller manager for AWS
```

Comment:
Does this table need an openstack entry?

Reply:
yes! thank you